docs/sections/contributor_guide/developer_setup.rst

```diff
@@ -65,10 +65,10 @@ As an alternative to installing `a prebuilt package from anaconda <https://anaco
 
 .. code:: sh
 
- # Activate your conda
- git clone https://github.com/maddenp/condev.git
- make -C condev package
- conda install -y -c local condev
+  # Activate your conda
+  git clone https://github.com/maddenp/condev.git
+  make -C condev package
+  conda install -y -c local condev
```

Comment:
Tweaked whitespace again, and this time added related advice to a nascent _Documentation_ page, blelow.